### PR TITLE
Add a build for OpenShift

### PR DESCRIPTION
**Pull request by etsauer**
@sabre1041 @raffaelespazzoli

### CODE REVIEW

File: build/build.yml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  spec:
+    failedBuildsHistoryLimit: 5
+    nodeSelector: null
+    output:
+      to:
+        kind: ImageStreamTag
+        name: cert-operator:latest
+    source:
+      git:
+        uri: https://github.com/etsauer/cert-operator
```
Comment (sabre1041) [inline build/build.yml]: Can the `uri` and `ref` point to the upstream repository?

**Comment by etsauer** [comment]
@sabre1041 added a deployment.

File: README.adoc [hunk 2]
```diff
@@ -21,18 +21,19 @@ dep ensure
...
 == Deployment to OpenShift
 
 [source,bash]
 ----
-operator-sdk build quay.io/etsauer/cert-operator
-docker push quay.io/etsauer/cert-operator
-oc apply -f deploy/rbac.yaml -n operators
-oc apply -f deploy/operator.yaml -n operators
+oc process -f build/build.yml | oc apply -f-
+oc process -f deploy/rbac.yaml | oc apply -f-
```
Comment (sabre1041) [inline README.adoc]: This requires a namespace parameter

**Comment by etsauer** [comment]
@sabre1041 addressed.

**Pull request merged**